src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -24,4 +26,13 @@ public static boolean isEven(int a) {
     public static boolean isOdd(int a) {
         return !isEven(a);
     }
+
+    public static int countEven(int[] array) {
+        Preconditions.checkNotNull(array, "Your array is null!");
```

Comment:
It's not a strict requirement, but I think it's better to have a static import of the checkNotNull method here. One reason is that other developers from our team have already used the static import in their pull requests. So I think we should have a consistency here